joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/XConfigFileCreationPass.scala

```diff
@@ -71,6 +73,18 @@ abstract class XConfigFileCreationPass(cpg: Cpg, private val rootDir: Option[Str
     file.absolutePathAsString.endsWith(pathEnd)
   }
 
+  protected def isExcludedFile(rootPath: String, filePath: Path): Boolean = {
+    val isAcceptedFile = SourceFiles.filterFile(
+      filePath.toAbsolutePath.toString,
+      rootPath,
+      Option(config.defaultIgnoredFilesRegex),
+      Option(config.ignoredFilesRegex),
+      Option(config.ignoredFiles)
+    )
+
+    !isAcceptedFile
+  }
```

Comment:
I would not invert the logic and stick to the `isAccepted` logic of `filterFile`. Just easier to comprehend. But i can live with the current version as well.